src/js/components/SearchBar/InputAction.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+import {darken} from "polished"
+import styled from "styled-components"
+
+import type {Styled} from "../../types/styled"
+import {cssVar} from "../../lib/cssVar"
+
+const InputAction: Styled<> = styled.button`
```

Comment:
Used for the buttons on the right side of the search input.

<img width="158" alt="image" src="https://user-images.githubusercontent.com/3460638/91352828-42843b80-e79f-11ea-8e9b-a4e6c3ece837.png">